### PR TITLE
Add DistributedTransactionAdminService to Scalar DB Server

### PR DESCRIPTION
**Pull request by brfrn169**
This PR adds `DistributedTransactionAdminService` to Scalar DB Server. Please take a look!

### CODE REVIEW

File: rpc/src/main/proto/scalardb.proto [hunk 1]
```diff
@@ -451,3 +451,55 @@ message TwoPhaseCommitTransactionResponse {
     Error error = 4;
   }
 }
+
+// DistributedStorageAdmin
+
+service DistributedTransactionAdmin {
```
Comment (brfrn169) [inline rpc/src/main/proto/scalardb.proto:457]: This is a service definition of the `DistributedTransactionAdmin` service.

File: core/src/main/java/com/scalar/db/transaction/rpc/GrpcTransactionAdmin.java [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.concurrent.ThreadSafe;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@ThreadSafe
+public class GrpcTransactionAdmin implements DistributedTransactionAdmin {
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/transaction/rpc/GrpcTransactionAdmin.java:45]: This is a gRPC client for the `DistributedTransactionAdmin` service.

File: server/src/main/java/com/scalar/db/server/DistributedTransactionAdminService.java [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+import com.scalar.db.util.ThrowableRunnable;
+import io.grpc.Status;
+import io.grpc.stub.StreamObserver;
+import java.util.Set;
+import javax.annotation.concurrent.ThreadSafe;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@ThreadSafe
+public class DistributedTransactionAdminService
```
Comment (brfrn169) [inline server/src/main/java/com/scalar/db/server/DistributedTransactionAdminService.java:36]: This is a gRPC server for the the `DistributedTransactionAdmin` service.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**